dotCMS/src/test/java/com/dotmarketing/filters/VanityUrlRequestWrapperTest.java

```diff
@@ -0,0 +1,100 @@
+package com.dotmarketing.filters;
+
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.utils.URLEncodedUtils;
+import org.junit.Test;
+import com.dotcms.mock.request.MockHttpRequest;
+import com.dotcms.mock.request.MockParameterRequest;
+import com.dotcms.vanityurl.model.VanityUrlResult;
+import com.google.common.collect.ImmutableMap;
+
+public class VanityUrlRequestWrapperTest {
+
+    final String URL = "URL";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5391667)